src/components/CheckoutPriceBox/CheckoutPriceBoxStyled.js

```diff
@@ -73,3 +74,12 @@ export const StyledPriceWrapper = styled.li`
     width: 100%;
   }
 `;
+
+export const CouponNoteStyled = styled.div`
```

Comment:
let's add a custom class name here, so publishers will be able to easily change the color of that text

Reply:
just added